lib/display/display.ts

```diff
@@ -27,6 +27,42 @@ export function displaySignatures(scanResults: ScanResult[]): string[] {
   return result;
 }
 
+export function displaySkippedFiles(
+  scanResults: ScanResult[],
+  options?: Options,
+): string[] {
+  const showSkippedPaths =
+    (options && options['print-skipped-unmanaged-files']) || false;
+
+  if (!showSkippedPaths || scanResults.length == 0) {
+    return [];
+  }
+
+  const result: string[] = [chalk.whiteBright('Skipped files:\n')];
+
```

Comment:
and that it's 10 instead of 15, 50 or 100 is just a coincident, it's just a random number. If we want to keep a limit, then feel free to suggest another number.